mitodl/mitxonline
# add course certificate migration

#### Pre-Flight checklist

  
#### What are the relevant tickets?
https://github.com/mitodl/mitxonline/issues/705

#### What's this PR do?
It's to migrate the course certificate from MM for DEDP courses. 
There are a couple of other changes:
 - Modify the `import_micromasters_data` command to accept argument `--num` so it runs a specific file
 - Creates a new mapping table for `CourseRun` to  'current' `PageRevision` (It will be populated outside PR)


#### How should this be manually tested?

- Run migration to generate mapping table

- Go to /admin/micromasters_import/courseruncertificaterevisionid/ to add some records for CourseRun -> PageRevision mapping. (This requires setup Certificates in CMS)

- Run migration script 
   (It requires a foreign data wrapper created locally see #859)
```./manage.py import_micromasters_data --num 009```

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)
